express/parsers/molecule.py

```diff
@@ -9,60 +10,70 @@
 import logging
 import tempfile
 
+
 class MoleculeParser():
     """
     Molecule parser class.
 
     Args:
-        args (list): args passed to the parser.
-        kwargs (dict): kwargs passed to the parser.
-            structure_string (str): structure string.
-            structure_format (str): structure format, poscar or espresso-in.
+        structure_string (str): structure string.
+        structure_format (str): structure format, poscar or espresso-in.
     """
 
-    def __init__(self, structure_string):
+    def __init__(self, structure_string: str, structure_format: str):
         self.structure_string = structure_string
+
+        if structure_format == "poscar":
+            self.ase_format = "vasp"
+        else:
+            self.ase_format = structure_format
+
         self.inchi_long, self.inchi = self.get_inchi()
 
-    def create_pdb_from_poscar(self):
+    def create_rdkit_molfrom_structure(self) -> rdkit.Chem.Mol:
```

Comment:
This function is a getter really, so let's either define it as a property or call `get_rdkit_mol` it is implied that the Mol will be from structure